.vib/keycloak/goss/goss.yaml

```diff
@@ -1,15 +1,9 @@
 file:
- /opt/bitnami/keycloak/standalone/configuration:
+ /opt/bitnami/keycloak/conf:
    exists: true
    filetype: directory
    mode: "0775"
    owner: root
-/opt/bitnami/keycloak/domain/configuration/application-users.properties:
```

Comment:
Please note that the minimum number of goss tests is 5. Could you please replace this one with its equivalent in the new major or a similar one?

Reply:
I have added a new one, could you check if it is valid?